### PR TITLE
add no-unused-vars, fix code

### PR DESCRIPTION
**Pull request by omonk**
Codebase had a bunch of unused vars everywhere, this PR changes this and adds an eslint rule to make sure it doesn't happen

### CODE REVIEW

File: src/components/Footer/Office.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const Office = ({ name, telephone, email, streetAddress }) => (
   <PaddedCol width={[1, 1, 1, 1 / 2, 1 / 2, 1 / 4]}>
     <Subtitle reverse>{name}</Subtitle>
     <Padding top={1} bottom={1}>
-      {streetAddress.map((address, i) => (
+      {streetAddress.map(address => (
         <StyledBodyPrimary key={address}>{address}</StyledBodyPrimary>
```
Comment (jsms90) [inline src/components/Footer/Office.js]: Better to change this to `key={i}`?

File: src/functions/meetup.js [hunk 2]
```diff
@@ -260,7 +261,6 @@ exports.handler = async (event, context, callback) => {
       console.log(
         `Not prod so not creating contentful event for ${meetup.eventName}`
       )
-      return
```
Comment (jsms90) [inline src/functions/meetup.js]: is this not needed?

File: src/components/Footer/Office.js (hunk 1 above)
Comment (omonk) [reply src/components/Footer/Office.js]: Using a map index arg as a key isn't a great pattern

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-array-index-key.md

Comment (omonk) [reply src/components/Footer/Office.js]: address is unique enough here and its not dynamic so it'll be ok

File: src/functions/meetup.js (hunk 2 above)
Comment (omonk) [reply src/functions/meetup.js]: Not really, we're not using the return value from this iteration

File: src/components/Footer/Office.js (hunk 1 above)
Comment (jsms90) [reply src/components/Footer/Office.js]: kk

**Pull request merged**